utils/dataconvert/dataconvert.cpp

```diff
@@ -111,7 +111,7 @@ void number_int_value(const string& data,
                       const datatypes::SystemCatalog::TypeAttributesStd& ct,
                       bool& pushwarning,
                       bool noRoundup,
-                      T& intVal)
+                      T& intVal, bool* saturate)
```

Comment:
I wonder why do you choose ptr and not ref here?

Reply:
Default assignment to a non-const ref parameter is not possible (default assignment so as to avoid changing all occurrences of this function called), that's why I am using a ptr instead.